framework/packages/abstract-sdk/src/base/endpoints/ibc_callback.rs

```diff
@@ -5,7 +5,7 @@ use cosmwasm_std::{Addr, Deps, DepsMut, Env, MessageInfo, Response};
 /// Trait for a contract's IBC callback ExecuteMsg variant.
 pub trait IbcCallbackEndpoint: Handler {
     /// Queries the IBC Client address.
-    fn ibc_client(&self, deps: Deps) -> Result<Addr, Self::Error>;
+    fn ibc_client(&self, deps: Deps) -> Result<Addr, AbstractSdkError>;
```

Comment:
Can't we keep Self::Error here?

Reply:
No because Self::Error is for the app developper, not the person implementing IBCCallbackEndpoint and Self::Error implements From<AbstractSdkError>, so it's better to keep AbstractSdkError here.

Reply:
Got mixed up, yes we can !